mkdocs_puml/plugin.py

```diff
@@ -61,6 +59,11 @@ def on_config(self, config: Config) -> Config:
             Full config of the mkdocs
         """
         self.puml = PlantUML(self.config['puml_url'], num_workers=self.config['num_workers'])
+        self.div_class_name = self.config['puml_keyword']
```

Comment:
`self.div_class_name` should always be `puml`. This is the class we set in the resulting html tag. We can apply various css styles (or fulfill other manipulations of the tag) using this class name.

You need to create new class attribute, for example, `self.puml_keyword` and set the value of the config.

Reply:
Again misunderstanding on my end, thanks for clarification, I will rework it!